src/pages/IndividualArticle/IndividualArticle.jsx

```diff
@@ -1,58 +1,87 @@
 import { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
-import { getIndividualArticle } from "../../api";
-import voteIcon from '../../assets/vote.svg';
+import { getCommentFromArticle, getIndividualArticle } from "../../api";
+import voteIcon from '../../assets/vote-icon.svg';
 import "./individualArticle.css"
 
 export default function SingleItemPage() {
   
-    const {article_id} = useParams();
-    const [article, setArticle] = useState({});
-    const [isLoading, setIsLoading] = useState(true);
-   
-    useEffect(() => {
-        setIsLoading(true);
-        getIndividualArticle(article_id)
-        .then((data) => {
-            console.log(data)
-            setArticle(data)
-            setIsLoading(false)
-        })
-    }, [])
+  const { article_id } = useParams();
+  const [article, setArticle] = useState({});
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [showComments, setShowComments] = useState(false); 
 
-    if(isLoading) {
-        return (
-          <div>
-            <h2>Loading...</h2>
-        </div>
-        )
-    }
+  function handleComments() {
+    setShowComments(!showComments);
+  }
+
+  useEffect(() => {
+    setIsLoading(true);
+    getIndividualArticle(article_id).then((data) => {
+      setArticle(data);
+      setIsLoading(false);
+    });
+  }, [article_id]);
 
+  useEffect(() => {
+    getCommentFromArticle(article_id).then((data) => {
```

Comment:
The comment should also have its own loading state to provide visual feedback to the user